misc/incremental_checker.py

```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/env python3
+"""
+This file compares the output and runtime of running normal vs incremental mode
+on the history of any arbitrary git repo as a way of performing a sanity check
+to make sure incremental mode is working correctly and efficiently.
+
+It does so by first running mypy without incremental mode on the specified range
+of commits to find the expected result, then rewinds back to the first commit and
+re-runs mypy on the commits with incremental mode enabled to make sure it returns
+the exact same result despite the files continuously changing.
+
+This script will by default, download and test mypy's repo. So, doing:
```

Comment:
Missing comma: "*,* by default,".  Alternatively, you could remove the comma after "by default".